content/docs/practices/naming.md

```diff
@@ -75,13 +75,13 @@ The list is not exhaustive.
 | Family | Base unit | Remark | 
 | -------| --------- | ------ |
 | Time   | seconds   |        |
-| Temperature | celsius | Celsius is the most common one encountered in practice |
+| Temperature | celsius | _celsius_ is preferred over _kelvin_ for practical reasons. |
```

Comment:
I'd say more than "practical reasons"

I'd leave it as Celsius rather than _celsius_ as we're talking about the unit names rather than the string in the metric name.

Reply:
The string in the metric name would be `celsius`. I used italics here because it's a verbatim reproduction of the entry in the "Base unit" column. I'm happy to capitalize, but then we should do in in the base unit column, too.

Reply:
What other reasons would you mention here?

Reply:
The reason is related to the concepts of Celsius and Kelvin, not the strings `celsius` and `kelvin` though.

The existing one is what I'd use, Celsius is much more common than Kelvin

Reply:
Again, I'm not using `celsius` and `kelvin`. I'm using _celsius_ and _kelvin_ exactly because I'm referring to units, _not_ to strings. With capitalization, that would be kind of clear as well, but then we need to capitalize in the _base unit_ column as well for consistency. Do you want me to do that?

I would not needlessly go down the rabbithole of discussing what unit is more common. I wrote a master thesis and a PhD thesis and read gazillions of papers that all used _K_ and never _C_. 300 million US Americans will happily swear that _F_ is most common.

I think the true reason is that most uses of Prometheus metric for temperature is things like CPU temperature or even temperature of cat beds, where the numbers get really high if we used _K_. Also, _C_ is probably what most sensors give you readily. But again, I wouldn't go down the rabbithole of discussing it scientifically. That's why staying vague here is most productive. We essentially only want to signal that we thought about _K_, to avoid the next bug report telling us that _K_ is the SI base unit.

Reply:
> I'm using celsius and kelvin exactly because I'm referring to units, not to strings. 

Oh sorry, my mistake. I had mis-parsed the markdown.